### PR TITLE
adding rescue on 503 for es checks

### PR DESCRIPTION
**Pull request by majormoses**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [x] Update README with any necessary configuration snippets

- [x] Binstubs are created if needed

- [x] RuboCop passes

- [x] Existing tests pass 

#### Purpose
To allow better handling of a 503 when cluster is having issues.

### CODE REVIEW

File: sensu-plugins-elasticsearch.gemspec [hunk 1]
```diff
@@ -42,6 +42,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'elasticsearch',     '1.0.14'
   s.add_runtime_dependency 'aws-es-transport',  '~> 0.1'
   s.add_runtime_dependency 'sensu-plugin',      '~> 1.2'
+  s.add_runtime_dependency 'json',      '~> 1.8'
```
Comment (sstarcher) [inline sensu-plugins-elasticsearch.gemspec]: why was this added?

Comment (majormoses) [inline sensu-plugins-elasticsearch.gemspec]: when I ran bundle I got it wanting to install json 2.x which is not compatible with the latest version of httparty: https://rubygems.org/gems/httparty
